tests/server-integration/CopyAndRename/RenameTest.hs

```diff
@@ -0,0 +1,68 @@
+-- SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+module CopyAndRename.RenameTest where
+
+import CopyAndRename.Common
+import Data.Aeson.QQ.Simple (aesonQQ)
+import Data.Functor ((<&>))
+import Test.Tasty.HUnit
+import Utils
+
+unit_rename :: IO ()
+unit_rename = cofferTest do
+  testCopyOrRename False
+
+  value <- viewRoot <&> scrubDates
```

Comment:
I think it's not necessary to do that. AFAIR we even don't check changes in `golden` tests in that way

Reply:
ok

Reply:
Hmmm, I think we could actually check these things in the Haskell tests:

* `set-field`: updates the entry's and the field's modified date
* `tag`, `copy`, `rename`: updates the entry's modified date, but not the fields
* `delete-field`: updates the entry's modified date
* `create`: sets the entry's and all the field's modified dates to the _same_ date
* all these endpoints set the new modified dates to a "recent-ish" date (see example below)

These were not tested in the golden tests because, well, they're hard to test with `bats` :sweat_smile: 

But here, it's much simpler to do things like this with `lens-aeson` (pseudocode):
```hs
t0 <- getCurrentTime
response <- createEntry "/a/b" [aesonQQ| { "fields": { "field1": ... } } |]
t1 <- getCurrentTime

let entryModifiedDate = responseBody response ^?! key "dateModified" . _JSON @_ @UTCTime
assertBool $ entryModifiedDate >= t0
assertBool $ entryModifiedDate <= t1

responseBody response ^?! key "fields" . key "field1" . key "dateModified" . _JSON @_ @UTCTime
  @?= entryModifiedDate



```